### ISSUE TITLE
skype meeting links in calendar events with 64 bit firefox

### ISSUE THREAD

**Issue by BGMcoder** [Ericsson/exchangecalendar#569]
Recently I switched to the 64 bit Firefox (version 51.0.1)

Normally, to join a meeting in my Skype for Business client, I click on the link in the calendar event and it opens the link in my browser and somehow the browser uses the skype web-app to launch my Skype for Business desktop client and the meeting is joined.

However, since I upgraded to 64bit, I no longer have a skype for business web-app to launch my client and I'm stuck.  I have to click the link, launch the url in my 64 bit browser and then copy the link, then open a 32 bit browser and past the url there.  Then it launches.

Now, I'm posting here because if I could just get the link from the calendar event I could paste it wherever I wanted.  But the links in the Desription area of the Exchange event don't permit Thunderbird's normal link context menu - so I don't have access to "copy link location" like I normally do.

So, I'm sort of requesting that y'all allow the normal link context menu to work in the description area of the calendar event.

**Comment by freechelmi** [comment]
Yes this is the lync server non sense ( not realted to this project) , you should fake the UA to be a 32 bits browser.

**Comment by BGMcoder** [comment]
Ah - I never thought of that.  So I installed a user-agent switcher from mozilla addons (https://addons.mozilla.org/en-US/firefox/addon/user-agent-switcher-revived/) and made sure I had an up-to-date user-agent string for a 32 bit setup.  However, that doesn't help anything.

I still wish I could just copy the url link from the calendar popup.
